.dockerignore

```
@@ -21,6 +21,7 @@ script/node_modules
 **/*.log
 _build/
 */_build/
+cli/bin/*.exe
```

Should also exclude the bc.js stuff
```suggestion
cli/bin/*.exe
cli/bin/*.bc.js
```